### DISCUSSION TITLE
Adding a partial book rating/review system...

### DISCUSSION THREAD

**Discussion by emigrating** [advplyr/audiobookshelf#2329] [Ideas]
Ideally ABS should be able to provide reommendations on other books (and podcasts) once you've finished a title or even if you're just looking for another listen.

The first step towards an algorithm would be some kind of rating system. I am partial to the way Audible does it, a star based system from 1-5 (or 0 for no rating at all) on the performance, content and overall title. This allows a person to love a certain book but absolutly loathe the narrator(s) (or vice versa) and the rate the overall package as mediocre because of it.

Thoughts?

**Comment by MikeGold** [comment]
It would be great to be able to rate books from 0 to 5 stars (preferably if other users of my audiobookshelf could see those ratings as well). If we could leave a short comment as well, it would be even better. I'm listening to a lot of books and being able to see my past comments on the books I've finished or the comments of my friends and family that listen to books on my server would be really nice.

**Reply by kalenio** [reply]
That's a great idea. Right now I'm curating a "recommended books" collection for other users to see, but there's not any nuance there. A 0-5 star rating system with short comments would be a much more robust solution.
